rtt-target/src/rtt.rs

```diff
@@ -74,8 +66,9 @@ impl RttChannel {
         ptr::write_volatile(&mut self.buffer, buffer as *mut u8);
     }
 
+    /// True if buffer NOT pointing to null
     pub fn is_initialized(&self) -> bool {
-        self.buffer.is_null()
+        !self.buffer.is_null()
```

Comment:
Huh, this is interesting. Could you send it as a separate PR?

Reply:
How could this even have worked before?

My assumption here is that its zero initialized. Thus `! ....is_null()` should mean that has been written to. And it works now, so I don't know what I'm missing here.

Reply:
I think this was changed just a couple of days ago and accidentally inverted

Reply:
Its just a "typo" fix, not sure it rewards a separate PR.